src/runtime/metal.cpp

```diff
@@ -195,6 +210,11 @@ struct module_state {
 };
 WEAK module_state *state_list = NULL;
 
+// API Capabilities.  If more capabilities need to be checked,
+// this can be refactored to something more robust/general.
+bool halide_metal_api_supports_set_bytes;
```

Comment:
These need to be WEAK and should not be prefixed with "halide_" as they are not exported. I'm guessing that if they are not WEAK, the runtime linker will insist that they are prefixed with "halide_". Is there a desire to export the variables or did you run into an issue with this? I can just merge it and make the fix afterward if that is expedient, but figured I'd check in re: intent and history first.